pages/_language/events/_event/index.vue

```diff
@@ -46,41 +46,14 @@
               <event-block-text-media v-bind="block" />
             </div>
             <div
-              v-if="block._modelApiKey === 'speakers_block'"
+              v-if="block._modelApiKey === 'chapters_block'"
               :key="block.id"
               class="event__layout event__layout--padded"
             >
-              <h3>
-                {{ block.title }}
-              </h3>
-
-              <ul class="list--inline">
-                <li
-                  v-for="speaker in block.speakers"
```

Comment:
Let's keep this for now?

Otherwise also remove the associated classes please.